ecommerce-ios/Extensions/Color/Color+Application.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Color+Application.swift
+//  ecommerce-ios
+//
+//  Created by Nguyen M. Tam on 13/05/2021.
+//
+
+import SwiftUI
+
+extension Color {
+
+    // MARK: - Gray
+
+    // #292A33 Gray 900
```

Comment:
Please add a variable comment for both colors here
```suggestion
    /// #292A33 Gray 900
```

Reply:
And all the colors below.